### PR TITLE
Add user facing changelog for `7.3.0`

### PR DESCRIPTION
**Pull request by jtpio**
Fixes https://github.com/jupyter/notebook/issues/7493

- [x] Add Notebook related changes
- [x] Add JupyterLab changes

### CODE REVIEW

**Comment by jtpio** [comment]
This should be good to go for a first pass, and we can iterate more on it if more changes land later.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,33 +12,58 @@ upgrading `notebook`.
...
+
+- a setting that can be defined by a user via the Settings Editor
+- the `View > Enable Full Width Notebook` menu entry
+- the command palette, searching for `Enable Full Width Notebook`
+
+![a screenshot showing how to enable full width notebook via the menu](https://github.com/user-attachments/assets/148b227a-874a-4bb3-a4c6-c7350525ccc4)
+
+This is particularly useful on large displays, or for those who prefer working with a wider notebook.
+
+It is also still possible to use the JupyterLab interface for a more fully-featured interface, which allows for full-width notebooks too, and offers a "Simple Interface" mode for only showing a single notebook at a time.
```
Comment (krassowski) [inline CHANGELOG.md:64]: Hard to read with three "interface" occurrences:

```suggestion
It is also possible to use the JupyterLab interface for a more fully-featured experience, which allows for full-width notebooks too, and offers a "Simple Interface" mode for only showing a single notebook at a time.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,33 +12,58 @@ upgrading `notebook`.
 Use `pip install pip --upgrade` to upgrade pip. Check pip version with
 `pip --version`.
 
-## v7.2
+## 7.3
```
Comment (krassowski) [inline CHANGELOG.md:15]: Worth adding a link to JupyterLab changelog and extension migration too as for 7.2?

Comment (jtpio) [reply CHANGELOG.md:15]: yes that makes sense :+1:

**Comment by jtpio** [comment]
Getting this one in so https://github.com/jupyter/notebook/pull/7481 can add more content to the changelog.

Happy to iterate more on the changelog until the final release.

**Pull request merged**